x-pack/plugins/endpoint/server/routes/resolver/queries/stats.test.ts

```diff
@@ -0,0 +1,188 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { EndpointAppConstants } from '../../../../common/types';
+import { StatsQuery } from './stats';
+import { fakeEventIndexPattern } from './children.test';
+
+describe('stats query', () => {
+  it('generates the correct legacy queries', () => {
+    expect(new StatsQuery('awesome-id').build('5')).toStrictEqual({
```

Comment:
❔ `awesome-id` I'm all for it,but is that OK for merge?

Reply:
it's an arbitrary string that is already used